docs/doc/14-sql-commands/00-ddl/101-network-policy/ddl-create-policy.md

```diff
@@ -0,0 +1,42 @@
+---
+title: CREATE NETWORK POLICY
+---
+
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.26"/>
+
+Creates a new network policy in Databend.
+
+## Syntax
+
+```sql
+CREATE NETWORK POLICY policy_name
```

Comment:
Support optional `[ IF NOT EXISTS ]`,
`CREATE NETWORK POLICY [ IF NOT EXISTS ] policy_name`